src/server.js

```diff
@@ -1,5 +1,6 @@
 import fs from 'node:fs'
 import express from 'express'
+import bodyParser from 'body-parser'
```

Comment:
body-parser is built-in now, btw: https://stackoverflow.com/questions/47232187/express-json-vs-bodyparser-json/47232318#47232318